rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -29,569 +29,550 @@
 import rocal_pybind as b
 from amd.rocal.pipeline import Pipeline
 
-def blend(*inputs,ratio=None):
-    kwargs_pybind = {"input_image0":inputs[0], "input_image1":inputs[1], "is_output":False ,"ratio":ratio}
-    blend_image = b.Blend(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+
+def blend(*inputs, ratio=None, rocal_tensor_layout=types.NHWC,
```

Comment:
Why only 3 arguments are present in line 33 ?
We can make all the arguments as a part of line 33 since a total of only 4 argumenst are present overall

Reply:
The inputs argument contains two rocal tensors, ratio, layout and output_type are present as arguments. is_output is hardcoded as False for all aug functions. So all pybind params are present as function arguments